runtime/runtime-standalone/contracts/cross-contract-high-level/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+RUSTFLAGS='-C link-arg=-s' cargo build --target wasm32-unknown-unknown --release
+cp target/wasm32-unknown-unknown/release/cross_contract_high_level.wasm ./res/
+wasm-opt -Oz --output ./res/cross_contract_high_level.wasm ./res/cross_contract_high_level.wasm
```

Comment:
you have it commented in another `build.sh`, so which version should be used (with or without the wasm-opt)?

Reply:
uncommented `wasm-opt` in status_message contract for a sake of consistency